Unable to generate from https://api.jellyfin.org/openapi/jellyfin-openapi-stable.json

With the following config:
```
package: main
generate:
  models: true
output: jellyfin.gen.go
```

Generating from https://api.jellyfin.org/openapi/jellyfin-openapi-stable.json at the time of writing

I get

```
/jellyfin.gen.go:886:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:887:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:888:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:889:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:890:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:891:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:892:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:893:62: invalid constant type TranscodingInfoTranscodeReasons
./jellyfin.gen.go:3263:18: VideoRangeType is not a type
./jellyfin.gen.go:5804:6: VideoRangeType redeclared in this block
        ./jellyfin.gen.go:731:2: other declaration of VideoRangeType
./jellyfin.gen.go:893:62: too many errors
```

I'll have a look into this at some point later, but I may have to use a different generator for now.

`TranscodingInfoTranscodeReasons` seems like it should be an enum but it's currently past midnight and I need to stop looking at this.

--- a/pkg/codegen/schema.go
+++ b/pkg/codegen/schema.go
@@ -970,7 +970,16 @@ func GenerateGoSchema(sref *openapi3.SchemaRef, path []string) (Schema, error) {
 		}
 
 		return outSchema, nil
-	} else if len(schema.Enum) > 0 || (globalState.is31 && schema.Const != nil) {
+	} else if (len(schema.Enum) > 0 || (globalState.is31 && schema.Const != nil)) && !t.Is("array") {
+		// An `enum` (or 3.1 `const`) is only meaningful on a scalar schema:
+		// it constrains a single value, which we render as a typed constant.
+		// A schema that combines `type: array` with an `enum` is malformed --
+		// the enum members are scalars, not arrays, so no array value could
+		// satisfy it (seen in the wild as a copy of the item enum onto the
+		// array; see issue #2176). Emitting constants there yields
+		// `const X SliceType = ...`, which is not a valid Go constant type.
+		// Fall through to normal array handling so the schema generates as a
+		// plain `[]ItemType`; the item schema carries the real enum.
 		err := oapiSchemaToGoType(schema, path, &outSchema)
 		// Enums need to be typed, so that the values aren't interchangeable,
 		// so no matter what schema conversion thinks, we need to define a

